src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -1035,6 +1043,9 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
       });
     }
 
+    // workspace enable and no workspace is selected
+    const isApplicationHome = workspaceEnabled && !workspaceId;
```

Comment:
Can we just name it `hideImport`? `isApplicationHome` too specific, it could be `workspaceEnabled && !workspaceId = true` but this component is not in application home

Reply:
updated